src/main/java/com/minecolonies/compatibility/tinkers/TinkersWeaponProxy.java

```diff
@@ -0,0 +1,32 @@
+package com.minecolonies.compatibility.tinkers;
+
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import slimeknights.tconstruct.library.tools.SwordCore;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'slimeknights.tconstruct.library.tools.SwordCore'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)